lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+class Menu
+
+  attr_reader :food
+
+  def initialize(food)
+    @food = food
```

Comment:
I love this approach. This way, you're not limited to always having the same menu in every instance of your Menu class, and you can instantiate it to whatever menu you want :)